docs/rtk-query/api/created-api/hooks.mdx

```diff
@@ -360,7 +360,10 @@ selectFromResult: () => ({})
 
 - **Returns**: A tuple containing:
   - `trigger`: A function that triggers an update to the data based on the provided argument. The trigger function returns a promise with the properties shown above that may be used to handle the behavior of the promise
-  - `mutationState`: A query status object containing the current loading state and metadata about the request, or the values returned by the `selectFromResult` option where applicable
+  - `mutationState`: A query status object containing the current loading state and metadata about the request, or the values returned by the `selectFromResult` option where applicable.
+    Additionally, this object will contain
+    - a `reset` method to reset the hook back to it's original state and remove the current result from the cache
```

Comment:
❓ 

Based on the description, I would have assumed that `originalArgs` gets reset as well. Should that be the case?

Otherwise, is it worth pointing out that it doesn't get reset?

Reply:
With the current implementation they will get reset.